wcivf/apps/people/models.py

```diff
@@ -189,12 +189,29 @@ def linkedin_username(self):
     @property
     def youtube_username(self):
         youtube_url = self.youtube_profile
-        if ("channel") in youtube_url:
+        if "channel" in youtube_url:
             return self.name + "'s Channel"
-        else:
-            youtube_split = list(filter(None, youtube_url.split("/")))
-            youtube_username = youtube_split[-1]
-            return youtube_username
+        youtube_split = list(filter(None, youtube_url.split("/")))
+        youtube_username = youtube_split[-1]
+        return youtube_username
+
+    @property
+    def long_statement(self):
+        return self.statement_count > 10
+
+    @property
+    def statement_count(self):
+        statement = self.statement_to_voters
+        statement_count = len(statement.split())
+        return statement_count
+
+    @property
+    def statement_intro(self):
+        statement_to_voters = self.statement_to_voters
+        statement_split = statement_to_voters.splitlines()
```

Comment:
I think we should use the same method to split here as above - but I don't know if splitting on space in both places is better than splitting on newlines.

The reason for this is: Imagine someone has a profile longer than 10 (or whatever) words, but all on one line. This will trigger the "long statement" interface, but if there's no extra lines, it won't show any extra content.

Maybe we give everyone a single line always, and hope they don't just write a single really really long line?

Or we split the text after so many words or a newline, whatever comes first?

Reply:
Good point. I tried a few ideas based on your suggestions and I thought the first sentence intro followed by full statement (less first line) worked well.